website/content/gateway/getting_started/signing_requests/signing_requests.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Signing_requests"
+date: 2022-04-12T14:29:19+02:00
+anchor: "signing-requests"
+weight: 35
+---
+
+## Signing requests
+
+POST requests must be signed. Other requests can optionally be signed.
+
+The signature is an RSA signature of the HTTP body; it is represented in hex. The partner account is identified by the partner’s signing API key. Both the partner’s signing API key and the RSA signature must be provided in a {{% highlight_text %}} Signature {{% /highlight_text %}}  header together with {{% highlight_text %}} RS256-hex {{% /highlight_text %}}:
+```
+Signature: <partner-signing-api-key> RS256-hex <signature>
+```
+
+{{% notice %}}{{% bold %}}Notice{{% /bold %}}: If the signature header is included, it must hold a correct signature, otherwise the transaction will fail.{{% /notice %}}
```

Comment:
```suggestion
{{% notice %}}**Notice**: If the signature header is included, it must hold a correct signature, otherwise the transaction will fail.{{% /notice %}}
```

The nested shortcodes seem to break rendering for me. Using Markdown syntax for bold works fine.

Reply:
I notice that the issue stems from the `.bold` style from `src/css/docs.css` not being applied properly, which seems to occur, because of the `:` immediately after `}}`.

```suggestion
{{% notice %}}{{% bold %}}Notice:{{% /bold %}} If the signature header is included, it must hold a correct signature, otherwise the transaction will fail.{{% /notice %}}
```

The above also renders properly for me.

Reply:
Strange. I think we should utilize the `.md` format as much as possible and not creating our own `shortcodes` when they are already supported.

Reply:
I agree. I don't see the benefit of using the `bold` shortcode defined in `website/layouts/shortcodes/bold.html`:

```html
<div class="bold">
    {{ .Inner }}
</div>
```

over asterisks, which just amount to `<strong></strong>`, on the contrary.

However, I assume @Lassejoe did it for a reason?

Reply:
I can't really remember why I did it in the beginning, I think it was because of some rendering issues.
I also used an older version of Hugo then, so that might have been the case.
IIRC, the more recent files has also utilized markdowns own bold format `** **`